lib/src/cli/generate/generate_command.dart

```diff
@@ -0,0 +1,27 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:args/command_runner.dart';
+
+class GenerateCommand extends Command<void> {
+  @override
+  final String description = 'Generate realm model objects from prototypes';
+
+  @override
+  final String name = 'generate';
+
+  @override
+  FutureOr<void>? run() async {
+    // Ensure realm_generator has run (EXPENSIVE!)
```

Comment:
I moved it to a separate generate command for that reason, so you can run `dart run realm generate` instead of `dart run build_runner build`. This is not called on `dart run realm metrics`

Reply:
Arhh.. I see the comment is stale of course